Configuration/TypoScript/setup.txt

```diff
@@ -40,4 +40,8 @@ plugin.tx_oidc.mapping {
 plugin.tx_oidc_login {
     userFunc = Causal\Oidc\Controller\LoginController->login
     defaultRedirectPid =
+    # Additional URL parameters for the authorization URL of the identity server
```

Comment:
I'd suggest to add a (commented-out) sample parameter, to make it clear how it may be used, thus serves as "a documentation":

```
# Additional URL parameters for the authorization URL of the identity server
authorizationUrlOptions {
    #theme = dark
}
```

Reply:
good idea. I added login_theme as a sample parameter.